lib/screens/home/pages/stats/stats_page.dart

```diff
@@ -125,3 +131,26 @@ class _StatsPageState extends State<StatsPage> {
     );
   }
 }
+
+void _showTokenExpiredDialog(BuildContext context) {
+  showDialog(
+    barrierDismissible: false,
+    context: context,
+    builder: (context) {
+      return AlertDialog(
+        title: Text("Token expired"),
+        content: Text("Relogin to refresh the jwt token"),
```

Comment:
```suggestion
        content: Text("Your session has expired! Please login again to access the Mentorship System features."),
```
This text should not include jwt token in the message, since we don't want to expose implementation details to the user. If I did not implement this, I would not know what a JWT token was :P Can you change it for something else please? No need to test again after since its just this text. I am not sure about a good message, but perhaps for now something like: "Your session has expired! Please login again to access the Mentorship System features."